.github/workflows/build-image.yml

```diff
@@ -41,37 +28,105 @@ jobs:
         run: |
           echo "Building an image with the following tag:"
           echo $IMAGE_TAG
+          echo "Based off the following commit:"
+          echo $GITHUB_SHA
       - name: Checkout
         uses: actions/checkout@v2
       - name: Configure AWS Credentials
         uses: aws-actions/configure-aws-credentials@v1
         id: configure-aws-creds
+        with:
+          aws-access-key-id: ${{ secrets.ECR_AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.ECR_AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-west-2
+      - name: Login to Amazon ECR
```

Comment:
Do we need to log in to ECR here? It looks like this job is only building the image, not pushing it.

Reply:
yeah good point, lemme remove